solidity/scripts/lcl-initialize.js

```diff
@@ -5,11 +5,6 @@ const TokenStaking = artifacts.require(
   "@keep-network/keep-core/build/truffle/TokenStaking"
 )
 
-const {
-  TokenStakingAddress,
-  TBTCSystemAddress,
-} = require("../migrations/external-contracts")
-
```

Comment:
Same here, I think this will not going to work without those lines.

Reply:
Same as above